mlrun/utils/notifications/notification_pusher.py

```diff
@@ -114,11 +115,12 @@ def _should_notify(
 
         # if at least one condition is met, notify
         for when_state in when_states:
-            if (
-                when_state == run_state == "completed"
-                and (not condition or ast.literal_eval(condition))
-            ) or when_state == run_state == "error":
-                return True
+            if when_state == run_state:
+                if (
+                    run_state == "completed"
+                    and (not condition or ast.literal_eval(condition))
+                ) or run_state in ["error", "aborted"]:
+                    return True
```

Comment:
why not using run state constants here?

Reply:
Import loop, @Tankilevitch worth moving `mlrun.runtimes.constants` to `mlrun.common`